lib/Driver/ToolChains.cpp

```diff
@@ -1191,6 +1233,23 @@ addLinkSanitizerLibArgsForDarwin(const ArgList &Args,
       /*AddRPath=*/ shared, TC);
 }
 
+static void
+addLinkSanitizerLibArgsForWindows(const ArgList &Args,
+                                 ArgStringList &Arguments,
+                                 StringRef Sanitizer,
+                                 const ToolChain &TC,
+                                 bool shared = true
+                                 ) {
+  // Sanitizer runtime libraries requires C++.
+  Arguments.push_back("-lc++");
+  // Add explicit dependency on -lc++abi, as -lc++ doesn't re-export
+  // all RTTI-related symbols that are used.
+  Arguments.push_back("-lc++abi");
```

Comment:
Why the assumption on the library names here and the spelling for the argument?  Furthermore, Windows doesn't use `libc++abi` ever but rather vcruntime.